### ISSUE TITLE
Crash on boot with MultiROM

### ISSUE THREAD

**Issue by MazeChaZer** [ubports/ubuntu-touch-for-oneplus-one#26]
I tried to install stable 8 and the latest devel-proposed. I used the modifed version of MultiROM Manager. Both crashed upon booting, generating this log file: http://pastebin.com/PTbFjrhR

**Comment by mariogrip** [comment]
Thanks for reporting!

do you get any error from the multirom manager? or from recovery when it installs?

**Comment by mariogrip** [comment]
this should be fixed now

**Comment by MazeChaZer** [comment]
Thanks for looking at this, I currently can't test this but I will report back a few days from now

**Comment by mariogrip** [comment]
I'll close this for now, please reopen it if the problem still is there.

**Comment by MazeChaZer** [comment]
@mariogrip I removed all the existing Ubuntu Touch MultiROM images and created a new one based on devel-proposed 161. The crash doesn't happen anymore, but now, when I select Ubuntu Touch in the MultiROM bootloader menu, the device restarts twice and I'm back in the bootloader menu. Unfortunately no new log file was written.
I noticed that the AK kernel installed by the MultiROM manager is for Android 5.0, while I'm running Android 5.1.1. Could that be a problem? I had some trouble finding a suitable kernel to flash on my own.
I didn't upgrade the customized MultiROM manager app, do I have to do that?
